solutions/src/main/java/ru/sberbank/school/task09/CityKryoSerializer.java

```diff
@@ -0,0 +1,33 @@
+package ru.sberbank.school.task09;
+
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.Serializer;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import lombok.NonNull;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CityKryoSerializer extends Serializer<City>{
```

Comment:
GenericWhitespace &#39;&gt;&#39; should followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-107657196>*